### DISCUSSION TITLE
A problem when play audio or video

### DISCUSSION THREAD

**Discussion by 39daoliu** [otakuhqz/Omega#483] [General]
I found a serious problem with Neo launcher. It often crashed the app interface when I was playing audio or video, causing the phone to need to restart.

I am using Lineages OS 21 and this problem does not occur when using the system launcher

**Comment by machiav3lli** [comment]
- Which version of Neo Launcher are you using?
- Is it set as the default launcher and granted permission to access notifications?
- Also please provide a crashlog using [Scoop](https://www.f-droid.org/de/packages/taco.scoop/)
